src/promptflow-azure/pyproject.toml

```diff
@@ -42,7 +42,7 @@ python = ">=3.8,<3.9.7 || >3.9.7,<3.13"
 azure-core = ">=1.26.4,<2.0.0"
 azure-storage-blob = {extras = ["aio"], version = ">=12.17.0,<13.0.0"}  # add [aio] for async run download feature
 azure-identity = ">=1.12.0,<2.0.0"
-azure-ai-ml = ">=1.14.0,<2.0.0"
+azure-ai-ml = ">=1.14.0,<1.19.0"
```

Comment:
I was assuming you lock this in dev_requirements.txt

Reply:
good catch, lock in dev req should be better